conans/client/cmake.py

```diff
@@ -52,10 +53,11 @@ def _generator(self):
                 return base
 
         if operating_system == "Windows":
-            if compiler == "gcc":
-                return "MinGW Makefiles"
-            if compiler in ["clang", "apple-clang"]:
-                return "MinGW Makefiles"
+            non_msvc = True if compiler != "Visual Studio" else False
```

Comment:
I think that compiler can't ever be "Visual Studio", because this case is handled above